src/torchmetrics/image/fid.py

```diff
@@ -219,6 +223,7 @@ def __init__(
         )
 
         if isinstance(feature, int):
+            num_features = int(feature)
```

Comment:
should already be int
```suggestion
            num_features = feature
```